schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/ClinicalReportRD.avdl

```diff
@@ -27,13 +27,6 @@ protocol ClinicalReportsRD {
         */
         string interpretationRequestVersion;
```

Comment:
I may be being picky, but could we think of having generic names (ie: `id` instead of `interpretationRequestId`) in fields like this?